riak_test/tests/gc_tests.erl

```diff
@@ -206,3 +223,47 @@ verify_riak_object_remaining_for_bad_key(RiakNodes, GCKey, {{Bucket, Key}, UUID}
     lager:info("As expected, BAD manifest in GC bucket remains,"
                " stand off orphan manfiests/blocks: ~p", [Manifest]),
     ok.
+
+verify_partial_gc_run(CSNode, RiakNodes,
+                      {MegaSec0, Sec0, _},
+                      {MegaSec1, Sec1, _}) ->
+    Start0 = MegaSec0 * 1000000 + Sec0,
+    End0 = MegaSec1 * 1000000 + Sec1,
+    Interval = erlang:max(1, (End0 - Start0) div 3),
+    Starts = [ {Start0 + N * Interval, Start0 + (N+1) * Interval}
+               || N <- lists:seq(0, 3) ],
```

Comment:
When `Start0=0, End0=5`, then `Interval=1` and the last end option `Start0 + (3+1) * Interval` is  `4`. Because it is smaller than `End0`.

Simple alternative to cover precisely between `Start0` and `End0` is:

```
[ {Start0 + N * Interval, Start0 + (N+1) * Interval}
           || N <- lists:seq(0, 2) ] ++ [Start0 + 3 * Interval, End0]
```